.gitlab-ci-full-integration-template.yml

```diff
@@ -55,10 +55,27 @@ test:integration:$CI_NODE_INDEX:
     - sysctl -w fs.inotify.max_user_instances=1024
     - sysctl -w fs.file-max=600000
     - ulimit -n 524288
+    # Mantra uploader: pull the helper script from mender-qa once. It’s tiny (<1 KiB).
+    - |
+      curl -sSfL \
+        https://raw.githubusercontent.com/mendersoftware/mender-qa/074b590110e1793ada4acac7bb7b1add8fe40981/scripts/mantra_post_test_results \
```

Comment:
We should use `master`. Otherwise we will soon or later forget to update :)

Reply:
There are pros and cons. The issue with master is that when we change something things might break. Locking to the specific revision will ensure it works.